tests/test_estimators/test_loess.py

```diff
@@ -0,0 +1,118 @@
+import numpy as np
+import pytest
+from sklearn.pipeline import Pipeline
+
+from sklego.linear_model import LoessRegressor
+
```

Comment:
you might want to . check how we test other methods... we use sklearns testing methods as well as our own.